content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -181,23 +174,28 @@ dotnet.exe example.dll
 {{< partial name="apm/apm-containers.html" >}}
 </br>
 
-3. After having instrumented your application, the tracing client sends traces to `localhost:8126` by default.  If this is not the correct host and port change it by setting the below env variables:
+3. After having instrumented your application, the tracing client sends traces to `localhost:8126` by default.  If this is not the correct host and port, change it by setting these environment variables:
 
-`DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`.
+    - `DD_AGENT_HOST`
+    - `DD_TRACE_AGENT_PORT`
```

Comment:
I don't know how these got here, but `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` are optional settings that apply to all instrumentation and they are explained below in the configuration section. This section is only for environment variables required to enable automatic instrumentation.

Reply:
Related: the first point above regarding `apm_non_local_traffic: true` only applies then the Agent is running in a different host or container. It's better suited for the "Linux Docker container" section below.

Reply:
I can move the `apm_non_local_traffic: true` step to the Linux Docker container for sure.

The two settings were in the docs in this step previously -- they have not been added in this edit -- and I think the "if this is not the correct host and port" correctly describes when should be used and when not, right?

Reply:
@lucaspimentel - taking a look at how other languages handle the wording, this is how Java handles it today as well. Let's move forward with these items to keep the tracer docs aligned - if we think we can do better we can create smaller PR to fix.

Reply:
@jaycdave88: Java has `apm_non_local_traffic` and `DD_AGENT_HOST`/`DD_TRACE_AGENT_PORT`, under the [Containers](https://docs-staging.datadoghq.com/kari/update-dotnet-core/tracing/setup_overview/setup/java?tab=containers#configure-the-datadog-agent-for-apm) section.

Other tracer do not have a section for "Required environment variables." This is specific to .NET because `CORECLR_ENABLE_PROFILING`/`CORECLR_PROFILER` is the only way to attach a profiler.

Reply:
But yeah, moving `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` in a separate PR is fine since it wasn't added in this PR.